flake.nix

```diff
@@ -0,0 +1,131 @@
+{
+
+  description = "LiquidHaskell packages";
+
+  inputs = {
+    nixpkgs.url = github:NixOS/nixpkgs/nixos-22.05;
+    flake-utils.url = github:numtide/flake-utils;
+
+    liquid-fixpoint.url = github:plredmond/liquid-fixpoint/nix-flake; # TODO change to official repo after merge
```

Comment:
Fix this to point at the ucsd-progsys/liquid-fixpoint if/when https://github.com/ucsd-progsys/liquid-fixpoint/pull/582 is merged.